frontend/src/pages/LogOut/LogOut.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect, Fragment } from 'react';
+
+const Logout = () => {
+  const [loading, setLoading] = useState(true);
+
+  const handleLogout = e => {
+    e.preventDefault();
+
+    fetch('user/logout', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Token ${localStorage.getItem('token')}`
+      }
+    })
+      .then(res => res.json())
+      .then(data => {
+        console.log(data);
+        localStorage.clear();
+        window.location.replace('http://localhost:3000/login');
```

Comment:
```suggestion
        localStorage.clear();
        navigate('/login')
```
remove console logs and use navigate. Dont use location.replace unless you are opening another site